### DISCUSSION TITLE
Inaccurate docs regarding Lerna

### DISCUSSION THREAD

**Discussion by johncmunson** [yarnpkg/berry#4004] [Q&A]
Why do the docs just [gloss over Lerna](https://yarnpkg.com/features/workspaces/#yarn-workspaces-vs-lerna) like everything is good? It seems pretty clear from the issue below that Yarn 2 is in fact not compatible with Lerna.

https://github.com/lerna/lerna/issues/2449#issuecomment-1018753790

If the Yarn team disagrees, or there are some parts of Lerna that are compatible and some that are not, then there should be an integration guide added to the docs. A single paragraph though is not appropriate. Lerna is a very widely used tool and it is almost always used in conjunction with Yarn.

Or, if Lerna and Yarn 2 are just outright incompatible with each other, then the docs are very wrong and should be updated.

**Comment by Daniel15** [comment]
>  It seems pretty clear from the issue below that Yarn 2 is in fact not compatible with Lerna.

A comment on the issue you linked to seems to imply the opposite?

> as of Jan 12, 2022, only lerna bootstrap doesn't work with yarn 2/3.
> However, other parts of lerna (v4, that I use) work well.

In any case, this repo is for the v1 site, so I'm going to move this issue to the Yarn v2+ repo.
